### PR TITLE
Add sca test file

### PR DESCRIPTION
**Pull request by Junefen**
Defined this test file to support sca testing of virt-who, initially developed 4 cases as below, will add more when get new requirements.

```
test_vw_report_mapping_with_sca_enabled
test_vw_host_to_guest_association
test_guest_entilement_with_sca_enabled
test_hypervisor_in_web
```

### CODE REVIEW

**Comment by Junefen** [comment]
Don't merge this pr, need some updates after define the api for stage candlepin.

File: tests/sca/test_virtwho_sca.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+class VirtWhoSCATestCase(Testing):
+
+    @classmethod
+    def setUpClass(cls):
+        super(VirtWhoSCATestCase, cls).setUpClass()
+        register_config = cls.get_register_config(cls())
+        if 'satellite' in register_config['type']:
+            cls.satellite_sca_set(cls(), cls.ssh_host(cls()), register_config, enable=True)
+        else:
+            cls.stage_sca_set(cls(), cls.ssh_host(cls()), register_config, enable=True)
```
Comment (Junefen) [inline tests/sca/test_virtwho_sca.py:17]: Maybe it's better to define a function ```sca_set()``` for the both satellite and stage using. Will perfect it in further prs

File: tests/sca/test_virtwho_sca.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        assert (self.op_normal_value(data, exp_error=0, exp_thread=1, exp_send=1))
+        assert (self.vw_rhsm_associate(data, self.host_uuid, self.guest_uuid))
+        assert (self.vw_web_associate(self.host_name, self.host_uuid, self.guest_name, self.guest_uuid))
+
+    def test_guest_entilement_with_sca_enabled(self):
+        ret, output = self.runcmd(cmd='subscription-manager status', ssh=self.ssh_guest())
+        assert ('Content Access Mode is set to Simple Content Access' in output)
+
+    def test_hypervisor_in_web(self):
+        pass
```
Comment (Junefen) [inline tests/sca/test_virtwho_sca.py:71]: This function need more efforts to research, will complete it when time is free.

**Review by hkx303 — APPROVED** [review]
ack, good jobs!

**Pull request merged**